ar4_hardware_interface/launch/main.launch.py

```diff
@@ -79,14 +80,16 @@ def generate_launch_description():
             "--controller-manager-timeout",
             "60",
         ],
-        condition=IfCondition(include_gripper),
+        # condition=IfCondition(include_gripper),
```

Comment:
Why is this commented?

Reply:
I can remove this. But I can't test if this still working.

Reply:
If we want to still support the configuration without gripper, this needs to be un-commented, if not then we can remove it, and probably remove all of the `include_gripper` conditions since we always will be including the gripper.

Reply:
We wil leave that permanent.